packages/vue/src/examples/cart/Cart.vue

```diff
@@ -130,7 +130,7 @@ export default {
             { name: "Size", value: "XS" },
             { name: "Color", value: "White" },
           ],
-          qty: "1",
+          qty: 1,
```

Comment:
The prop quantity has a type of string/number so it should work even when a prop passe dis a string "1". If it does not work, maybe the logic of SFCollectedProduct should be refactored. It should accept both prop types (number and string) and eventually convert string to number so that it adds it correctly